lib/smart_proxy_dynflow_core/launcher.rb

```diff
@@ -90,6 +90,19 @@ def https_app
       ssl_options |= OpenSSL::SSL::OP_NO_SSLv3 if defined?(OpenSSL::SSL::OP_NO_SSLv3)
       ssl_options |= OpenSSL::SSL::OP_NO_TLSv1 if defined?(OpenSSL::SSL::OP_NO_TLSv1)
 
+      if Settings.instance.tls_disabled_versions
+        Settings.instance.tls_disabled_versions.each do |version|
+          constant = OpenSSL::SSL.const_get("OP_NO_TLSv#{version.gsub(/\./, '_')}") rescue nil
```

Comment:
There was a potential issue, when the user would specify `1.1` instead of `'1.1'` in settings. I've added additional `to_s` there, just in case.